iam/policy-templates/dss-s3-copy-sfn-lambda.json

```diff
@@ -0,0 +1,42 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "logs:CreateLogGroup",
+        "logs:CreateLogStream",
+        "logs:PutLogEvents"
+      ],
+      "Resource": "arn:aws:logs:*:*:*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": "s3:*",
```

Comment:
This lambda shouldn't have delete access to the bucket. That's dangerous. Let's set the permissions on an as-needed basis.

Reply:
Done.